docs/how-to/extending/builtin.qmd

```diff
@@ -169,3 +172,60 @@ con.execute(cardinality("foo"))
 
 Here, Snowflake is informing us that the `ARRAY_SIZE` function does not accept
 strings as input.
+
+
+## Aggregate functions
+
+Let's the use the BigQuery backend to demonstrate how to access a BigQuery-only
```

Comment:
```suggestion
Let's use the BigQuery backend to demonstrate how to access a BigQuery-only
```

Reply:
No longer relevant because I switched this section to use DuckDB